operations/md_ops/pr_to_md.py

```
@@ -0,0 +1,376 @@
+from datetime import datetime
+from typing import Dict, List, Optional
+
+
+def format_datetime(dt_string: Optional[str]) -> str:
+    """Format datetime string to UTC format."""
+    if not dt_string:
+        return "N/A"
+    try:
+        dt = datetime.strptime(dt_string, "%Y-%m-%dT%H:%M:%SZ")
+        return dt.strftime("%Y-%m-%d %H:%M:%S UTC")
+    except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)